bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/scheduling/ReceiveField.java

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com)
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.runtime.internal.scheduling;
+
+/**
+ * A worker receive field for multiple receive action.
+ * <p>
+ * e.g., WRK_RECEIVE {w1 , w2};
+ *
+ * @since 2201.9.0
+ */
+public class ReceiveField {
+
+    private final String fieldName;
```

Comment:
I think we can make these public since those are internal package and remove getters.